server/models/handlers/accountCreation.js

```diff
@@ -0,0 +1,26 @@
+/* The goal: Take in the user input from the Account Creation form in 
+the front end UI in order to create a user account in the backend*/
+
+async function userAccount(firstName, lastName, type, studentID, email, password, phone) {
```

Comment:
I would recommend renaming this method to be `createUserAccount` since it would have the side effect of actually inserting the user into the database.

Reply:
Hi Amy,
Thank you for suggesting that!
**My action:** I've changed the name from userAccount to createUserAccount. 
You can now see these changes the following locations:
1.  line 14 of server/routes/publicAccountCreation.js
2. line 24 of server/test/accountCreation.test.js
3. line 4 of server/models/handlers/accountCreation.js

Reply:
Yes, that works. Thank you!